docs/configuration/version.rst

```diff
@@ -119,6 +119,35 @@ version::
 
 Input objects have same structure as deserialization closure inputs.
 
+.. _version-incrementing:
+
+Incrementing
+------------
+
+Incrementing phase does incrementing the version in accordance with *version incrementer* rule. By default version patch
+(least significant) number is incremented. There are predefined rules:
+
+- *incrementPatch* - increment patch number
+- *incrementMinor* - increment minor (middle) number
```

Comment:
There is an inconsistency between `incrementMinor` in a documentation and `incrementMinorVersion` in code. The same for `incrementPatch`.